src/app/core/entity-list/entity-list/entity-list.component.ts

```diff
@@ -108,12 +112,13 @@ export class EntityListComponent<T extends Entity>
 
   isDesktop: boolean;
 
-  listName = "";
-  columns: (FormFieldConfig | string)[] = [];
-  columnGroups: GroupConfig[] = [];
+  @Input() title = "";
+  @Input() columns: (FormFieldConfig | string)[] = [];
+  @Input() columnGroups: ColumnGroupsConfig;
+  groups: GroupConfig[] = [];
```

Comment:
I am not too happy with this naming change but I did not want to require a migration

Reply:
maybe we can collect (or then do a detailed review of) all config namings as part of the Admin UI work and then prepare one large, automatic migration to clean all of this up and make it fully consistent across different components.

Reply:
Lets collect them here https://github.com/Aam-Digital/ndb-core/issues/2066